mne/viz/backends/_ipyvolume.py

```diff
@@ -183,6 +187,7 @@ def quiver3d(self, x, y, z, u, v, w, color, scale, mode, resolution=8,
                  opacity=1.0, scale_mode='none', scalars=None,
                  backface_culling=False):
         color = np.append(color, opacity)
+        x, y, z = [_check_array(n) for n in [x, y, z]]
```

Comment:
`x, y, z = map(_check_array, [x, y, z])`

for once functional programming is nicer :)